spec/unit/batch_actions/resource_spec.rb

```diff
@@ -58,6 +58,11 @@
       expect(resource.batch_action_path).to eq "/admin/posts/batch_action"
     end
 
+    it "includes :scope and :q params" do
+      params = {q: {name_equals: 'Any'}, scope: :all}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.